src/models/timer.c

```diff
@@ -55,4 +55,14 @@ void updateTimers()
         }
 
     }
+}
+
+void resetTimers()
```

Comment:
maybe `clearTimers()` is a better name to avoid confusion with `resetTimer()`

Reply:
Yeaah i was trying to find a better function name but my brain just couldn't, so yep that's perfect to change to `clearTimers()`